src/builder/create_interaction_response.rs

```diff
@@ -278,6 +278,26 @@ impl CreateInteractionResponseMessage {
 /// They're same according to Discord, see
 /// [Autocomplete docs](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-response-object-autocomplete).
 pub type AutocompleteChoice = CommandOptionChoice;
```

Comment:
```suggestion
#[must_use]
pub type AutocompleteChoice = CommandOptionChoice;
```

We should probably make this `#[must_use]` since it gets sent off to the API.